Entity/Calendar.php

```diff
@@ -83,7 +83,7 @@ public function getStartDate()
      *
      * @return Calendar
      */
-    public function setStartDate(\DateTime $startDate)
+    public function setStartDate(\DateTime $startDate = null)
```

Comment:
because of NotBlank Validator. If you put anything in startDate input. Symfony form will populate the entity calendar with setStartDate(\DateTime $startDate) which is not possible.

If you see another way